kratos/input_output/vtk_output.cpp

```diff
@@ -1002,6 +1005,45 @@ void VtkOutput::ForceBigEndian(unsigned char* pBytes) const
 /***********************************************************************************/
 /***********************************************************************************/
 
+VtkOutput::GeometryType::Pointer VtkOutput::ReorderConnectivity(GeometryType::Pointer pGeometry) const
+{
+    const auto& r_geometry_type = pGeometry->GetGeometryType();
+    if (r_geometry_type == GeometryData::KratosGeometryType::Kratos_Hexahedra3D20) {
+        auto p_reorder_geom = Kratos::make_shared<GeometryType>();
+        GeometryType::PointsArrayType& r_reordered_points = p_reorder_geom->Points();
+        r_reordered_points.reserve(15);
```

Comment:
```suggestion
        r_reordered_points.reserve(pGeometry->PointsNumber());
```

to avoid bugs (like you currently have